cli/determined_cli/tensorboard.py

```diff
@@ -43,6 +44,10 @@ def start_tensorboard(args: Namespace) -> None:
         "trial_ids": args.trial_ids,
         "experiment_ids": args.experiment_ids,
     }
+
+    if args.context is not None:
+        req_body["user_files"] = context.read_context(args.context, constants.MAX_CONTEXT_SIZE)[0]
```

Comment:
non-blocking: We can drop the argument that's the same as the default value anyway, and I think multiple assignment makes a bit more sense than indexing here:
```suggestion
        req_body["user_files"], _ = context.read_context(args.context)
```